Week2/homework/js-exercises/drinkTray.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const drinkTypes = ['cola', 'lemonade', 'water'];
+
+let drinkTray = [];
+
+for( let i=0;i<5;i++) {
+        drinkTray.push(drinkTypes[i%drinkTypes.length]);
+    
+    }
+    console.log("Hey guys, I brought a " + drinkTray + ".");
```

Comment:
👍 .
Also try think of a way to do this without using the `%` operator, instead think about how to check for duplicate drinks in the `drinkTray` and add another drink if there are already 2 inside.